packages/react-notion-x/src/block.tsx

```diff
@@ -832,6 +832,37 @@ export const Block: React.FC<BlockProps> = (props) => {
         </components.pageLink>
       )
 
+    case 'table':
+      return (
+        <table className='notion-simple-table'>
```

Comment:
Can you add the notion blockId as a class? Like `className={cs(blockId)}`. You'll see most of the other blocks also have this. This is so people can target specific blocks and customize it with css.

Reply:
got it.just finished